tao/tao_bridge.bzl

```diff
@@ -23,6 +23,9 @@ def tao_bridge_cc_test(
             "@local_config_tf//:libtensorflow_framework",
         ],
         data = data,
-        linkstatic = linkstatic,
+        linkstatic = select({
+            "@local_config_blade_disc_helper//:is_mkldnn": 1,
```

Comment:
nit: It's not appropriate to hard code a specific logic about mkldnn here in a common rule, IMO. Just pass `linkstatic=1` for mkl-related tests?

Reply:
Currently `tao_bridge` does not have tests actually uses `mkldnn`. This is just for the linking problems.